action_msgs/GoalStatus status

nav_msgs/Goals target_poses

# 3. 数值类型直接映射
float64 start_time
float64 distance
float64 remain_distance